packages/saved-views-react/src/createViewState.ts

```diff
@@ -32,6 +32,29 @@ import {
   extractDisplayStyle,
   extractDisplayStyle3d,
 } from "./translation/displayStyleExtractor.js";
+import { Id64Array } from "@itwin/core-bentley";
+
+/**
+ * Settings for how to handle the visibility of elements (models or categories) in iModel.
+ *   * `enabled` – Enabled is the set of enabled elements (models or categories) that are stored in the saved view. Default is ignore.
+ *   * `disabled` – Disabled is the set of disabled elements (models or categories) that are stored in the saved view. Default is ignore.
+ *   * `other` – Other is the set of elements (models or categories) that are not stored in the saved view. Default is ignore.
```

Comment:
I would put these docs of each under each of the properties documents as well below so that when someone hovers over the setting, they can easily read the explanation